docs/api-reference/3d-tiles/overview.md

```diff
@@ -1,9 +1,17 @@
-# 3D Tiles Category
```

Comment:
We could rename the `overview.md` to `README.md`... Then their URLs will be simpler `https://loaders.gl/docs/api-reference/3d-tiles`.